docs/app/templates/docs/index.md

```diff
@@ -8,10 +8,54 @@ To install Ember Animated, run
 ember install ember-animated
 ```
 
+## Usage in Template Tag Format
+
+For usage in `gts` and `gjs`, `ember-animated` provides following exports from the index:
+
+```gjs
+import {
+  AnimatedBeacon,
+  AnimatedContainer,
+  animatedEach,
+  animatedIf,
+  AnimatedOrphans,
+  animatedValue,
+} from 'ember-animated';
```

Comment:
It might be nice to test these imports in the test app by creating some .gts tests?